internal/observability/call.go

```diff
@@ -496,6 +499,17 @@ func (c call) logStreamEvent(err error, success bool, succMsg, errMsg string, ex
 	ce.Write(fields...)
 }
 
+// emitSpanErrorTag sets the error information as tags on the current span
+func (c call) emitSpanErrorTag(res callResult) {
```

Comment:
Let's indicate in the name that this tags should be set only for application error.